README.md

```diff
@@ -1,2 +1,178 @@
-# Portfolio
-My Portolio project, 
+<a name="readme-top"></a>
+
+<div align="center">
+
+ <!-- LOGO -->
+
+  <img src="img/WazaCode Logo.png" alt="logo" width="auto"  height="auto" />
+  <br/>
+
+<!-- MAIN HEADING -->
+
+  <h3><b>My Portfolio</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- INTRO -->
+# 📖 Portfolio <a name="about-project"></a>
+
+> Here is a Portfolio project showcasing all my Abilities, Skills and Achievements.
+
+## 🛠 Built With <a name="built-with"></a>
+1. HTML
+2. CSS
+3. Linters
+### Tech Stack <a name="tech-stack"></a>
+
+> Planning to update it soon,
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">Linters</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> - Added Linters
+> - Added HTML
+> - Added CSS
```

Comment:
Kindly add features you introduced in this milestone such as ` added toolbar`, `added navigation` etc  so as to give a better explanation to users of the features you introduced in each milestone😄

Reply:
Ohh thank you